desktop_server/src/com/mygdx/game/server/ecs/entityfactory/TechnologyFactory.java

```diff
@@ -42,10 +42,10 @@ public TechnologyFactory(
   public int createEntity(@NonNull TechnologyConfig config, @NonNull Coordinates coordinates, Client client) {
     var entity = world.create();
 
-    var position = setUpCoordinates(coordinates, entity);
+    var newCoordinates = setUpCoordinates(coordinates, entity);
```

Comment:
ale po co te koordynaty są na serwerze xD

Reply:
ogólnie to wiem, że jest kiepski interfejs z tym entityFactory, ale to trzeba refactorować, a nie coś wymyślać

Reply:
refactor z ComponentFactor fajny, ale użyłbym go jako coś co jest używane w tych fabrykach, a z samych fabryk wywalił wspólny interfejs, bo chyba nie jest jakoś potrzebny?